### ISSUE TITLE
Avatars plugin: Avatar is displayed even if both BIMI and Gravatar are disabled

### ISSUE THREAD

**Issue by yasuhirokimura** [the-djmaze/snappymail#1240]
While investigating issue #1209 I faced strange behavior of Avatars plugin.

Currently I use Avatars plugin with following setttings.

1. Identicon: Name characters else silhouette
2. Enable "Preload valid domain icons"
3. Enable "Delay lookup"
4. Disable "BIMI"
5. Disable "Gravatar"

Since both BIMI and Gravatar are disalbed. I think avatar of each messages should be either name characters or silhouette. But if the message that come from `amazon.com`, `github.com`, `google.com`, etc. is accessed, then avatar for the domain is displayed. And if delay lookup setting is disabled, then either name characters or silhouette is displayed for such domains.

Is it expected behavior?

Browser: 64bit Google Chrome 116.0.5845.141 on 64bit Windows 10 22H2
IMAP daemon: Dovecot 2.3.20
PHP: 8.2.10
SnappyMail: 2.28.4
Mode: standalone

**Comment by the-djmaze** [comment]
> But if the message that come from `amazon.com`, `github.com`, `google.com`, etc. is accessed, then avatar for the domain is displayed.

Correct, because they are defined here: https://github.com/the-djmaze/snappymail/tree/master/plugins/avatars/images/services
